src/Lykke.AzureStorage/Tables/Paging/AzurePagingInfo.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Lykke.AzureStorage.Tables.Paging
+{
+    public class AzurePagingInfo : PagingInfo
+    {
+        public TableContinuationToken NextToken
+        {
+            get => _deserializeToken(NextPage);
+            set => NextPage = _serializeToken(value);
+        }
+
+        public List<TableContinuationToken> PreviousTokens
```

Comment:
Why not ReadOnlyList?

Reply:
Correct, probably it would be better to use interface instead.